libs/core/src/lib/fundamental-ngx-core.module.ts

```diff
@@ -142,6 +143,6 @@ import { MessageToastService } from './message-toast/message-toast-service/messa
         CardModule,
         MessageToastModule
     ],
-    providers: [AlertService, DialogService, NotificationService, MessageToastService]
+    providers: [AlertService, CarouselService, DialogService, NotificationService, MessageToastService]
```

Comment:
We don't need to include `CarouselDirective` since it's defined as provider on the CarouselDirective level.
Please correct me if I'm wrong.

Reply:
fixed now. Does not need service here.